teams/Aragon One/2019.md

```diff
@@ -21,7 +21,6 @@ Focus areas for this initiative:
 
 - ✅ **Custom labels**: Easily tag any Ethereum address on the client. Also easily share tags with other people via a file or a link.
 - 🔜 **Individual identity**: Allow individuals to create and manage their user profile, mapping their address to a human readable ENS name that can be used to interact with apps within the organization. [Autark](https://github.com/aragon/flock/blob/master/teams/Autark/2019Q1-2.md) is the one leading this initiative, and we will support via design and code reviews.
-- **Organization identity & membership support**: Allow users to create and manage their organizations’ profiles, providing an intuitive way to add members to a given organization and granting them permissions & privileges.
```

Comment:
Reason to remove: In the case of org's identity, Autark is taking take of it. In the case of org membership, we believe it's such an important piece, but we won't have design nor frontend bandwidth to think or implement it during Q4 2019.